pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -249,20 +249,28 @@ func (adc *attachDetachController) podDelete(obj interface{}) {
 	adc.processPodVolumes(pod, false /* addVolumes */)
 }
 
+func toNodeIdentifier(node *v1.Node) types.NodeIdentifier {
+	return types.NodeIdentifier{
+		Name: types.NodeName(node.Name),
+		ID:   node.UID,
+	}
```

Comment:
So afaict this creates `NodeIdentifier` with ID of `""` right? Where are we populating this ID with an actual value?